network_modeling/configs/templates/1.3/aruba/common/ncn-s.lag.j2

```diff
@@ -5,10 +5,11 @@ interface lag {{ node.config.LAG_NUMBER }} multi-chassis
     no shutdown
     description {{ node.config.DESCRIPTION }}
     no routing
-    vlan trunk native {{ variables.NATIVE_VLAN }}
 {%- if node.destination_port == 1%}
+    vlan trunk native {{ variables.NATIVE_VLAN }}
     vlan trunk allowed {{ variables.SPINE_LEAF_VLANS }}
 {%- else %}
+    vlan trunk native {{ variables.NATIVE_VLAN }}
```

Comment:
I believe this is incorrect.  Should be native 10 as you have in 1.4.

Reply:
good catch, fixed.